testdata/v2_full_yaml/terraform/envs/prod/datadog/main.tf

```diff
@@ -0,0 +1,2 @@
+# Auto-generated by fogg. Do not edit
+# Make improvements in fogg, so that everyone can benefit.
```

Comment:
I'm not sure I like this diff, I think this is not supposed to happen and I may have introduced a bug?